lib/addons/coredns/index.ts

```diff
@@ -0,0 +1,21 @@
+import { CfnAddon } from "@aws-cdk/aws-eks";
+import { ClusterAddOn } from "../..";
+import { ClusterInfo } from "../../stacks/cluster-types";
+
+export class CoreDnsAddOn implements ClusterAddOn {
+
+    version: string;
+
+    constructor(version?: string) {
+        this.version = version ?? "v1.8.0-eksbuild.1";
+    }
+    
+    deploy(clusterInfo: ClusterInfo): void {
+        new CfnAddon(clusterInfo.cluster.stack, "coredns-addon", {
```

Comment:
I see this block showing up multiple times. Can you create a generic managed addon implementation and derive from it? The reason why I bring it up:  there will be many managed addons in the future and we should provide a path for the customer to use them without waiting for a new build of SSP.